static/js/components/dashboard/CourseAction_test.js

```diff
@@ -241,14 +241,38 @@ describe('CourseAction', () => {
       course.runs.length > 0 &&
       course.runs[0].status === STATUS_WILL_ATTEND
     ));
-    let startDate = moment(now).add(10, 'days').toISOString();
-    let firstRun = alterFirstRun(course, {course_start_date: startDate});
+    let startDate = moment(now).add(10, 'days');
```

Comment:
i just made this change to match the pattern in the tests that i added below. also, it makes more sense for a variable called `startDate` to be a moment date rather than a string. there's no functional difference from this change